nextflow.config

```
@@ -59,6 +59,7 @@ params {
     // assembly options
     coassemble_group                     = false
     spades_options                       = null
+    spades_use_contigs_not_scaffolds     = false
```

```suggestion
    spades_use_contigs_not_scaffolds     = false
```

I think I would rather be more explicity: `--spades_outputtype contigs` with enum contigs or scaffolds (Scaffolds as default for backgfwards compatibility)

---

Changed accordingly